File: C/Assignments/Assignment 5/question_7.c
/*
Q7. Print a Floyd’s triangle pattern
Input: n = 4
Output:
1
2 3
4 5 6
7 8 9 10
*/

#include<stdio.h>
void main(){
    int n, count=1;
    printf("Enter a number: ");
    scanf("%d", &n);

    for(int i=0; i<5; i++){
        for(int j=0; j<=i; j++){
            printf("%d ", count);
            count++;
        }
        printf("\n");
    }
}

File: CtoJava/src/Assignment_5/question_7.java
package Assignment_5;

public class question_7 {
	/*
	Q7. Print a Floyd’s triangle pattern
	Input: n = 4
	Output:
	1
	2 3
	4 5 6
	7 8 9 10
	*/

	public static void main(String args[]){
	    int n=4, count=1;
	    

	    for(int i=0; i<n; i++){
	        for(int j=0; j<=i; j++){
	        	System.out.printf("%d ", count);
	            count++;
	        }
	        System.out.printf("\n");
	    }
	}
}
